src/components/Accordion/AccordionItem.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { AccordionItem as CarbonAccordionItem } from 'carbon-components-react';
+
+// same default used in carbon. adding to avoid prop type validation issues.
+const defaultRenderExpando = props => <button type="button" {...props} />;
```

Comment:
Small typo in variable name `Expando`

Reply:
Hey @sstone2423. Is the typo `Expando`? I think this is intentional?